Portfolio.html

```diff
@@ -3,12 +3,86 @@
 <head>
     <meta charset="UTF-8">
     <link href="https://fonts.googleapis.com/css?family=Quicksand" rel="stylesheet">
-    <link rel="stylesheet" type="text/css" href="Portfolio.css">
-    <title>Rob Reeves Portfolio</title>
+    <link href="https://fonts.googleapis.com/css?family=Fanwood+Text" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href=":css/normalize.css">
+    <link rel="stylesheet" type="text/css" href=":css/portfolio.css">
+    <link rel="stylesheet" type="text/css" href=":css/media-query.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Rob Reeves | Portfolio</title>
 </head>
-<body>
-
-
 
+<body>
+    <header class="main-nav">
+        <div class="nav-button-left">
+            <a class="nav-text" href="#C1">ABOUT ME</a>
+            <a class="nav-text" href="#C3">CONTACT</a>
+        </div>
+        <div class="nav-button-right">
+            <a class="nav-text" href="#C2">THREE PEDALS ONLY</a>
+        </div>
+    </header>
+        <div class="main-header-wrapper">
+            <div class="main-header-centre">
+                <h1>ROB REEVES</h1>
+                    <div class="sub-heading-wrapper">
+                        <div class="sub-heading">
+                            <h3>JUNIOR WEB DEVELOPER</h3>
+                        </div>
+                    </div>
+                    <div class="sub-heading-wrapper2">
+                        <p class="sub-heading2">There are lots of interesting things about me on this website. Please take your time and enjoy your stay. If you have any queries about anything then please don't hesitate to drop me a line.</p>
+                    </div>
+            </div>
+        </div>
+    <section class="main-content" id="C1">
+        <div class="main-content-wrapper">
+            <div class="h2-sub-heading">
+                <h2>ABOUT ME</h2>
+            </div>
+            <p class="main-content-text">I am a junior web developer with a passion for design and agile development. I am currently learning HTML, CSS, Javascript, PHP, and MySQL. I am also familiar with Git and other software environments. I am a certified scrum master and will be applying for my PHP Zend certification after I have graduated from Mayden Academy.</p>
+            <div class="article">
+                <img width="" height="" src=":img/a-drop-of-water-osa-water-leaf-158725.jpeg" class="image" alt="sea shore">
```

Comment:
remove width and height attributes if you dont need them

Reply:
fixed.